src/components/sidebar/FilterSidebar.jsx

```diff
@@ -72,24 +133,63 @@ const FilterSidebar = () => {
         <div className={classes['filter-type']}>
           <h2 className={classes['filter-type-title']}>Categories</h2>
           <div className="inp-container">
-            <input type="checkbox" name="category" id="category" checked />
-            <label for="">Men</label>
+            <input
+              onChange={categoryChangeHandler}
+              type="checkbox"
+              name="category"
+              id="Statement Edition"
```

Comment:
I don't think whitespaces should be used in `id` 

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id#:~:text=id%27s%20value%20must%20not%20contain%20whitespace%20(spaces%2C%20tabs%20etc.).%20Browsers%20treat%20non%2Dconforming%20IDs%20that%20contain%20whitespace%20as%20if%20the%20whitespace%20is%20part%20of%20the%20ID.%20In%20contrast%20to%20the%20class%20attribute%2C%20which%20allows%20space%2Dseparated%20values%2C%20elements%20can%20only%20have%20one%20single%20ID%20value.

Reply:
Done